src/structurizr/api/structurizr_client.py

```diff
@@ -109,14 +114,36 @@ def __enter__(self):
         return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
-        """Exit a context by unlocking the corresponding remote workspace."""
+        """Exit a context by unlocking the corresponding remote workspace.
+
+        Note: this method is deprecated in favour of lock(), and will be removed in a
+        future relesae.
```

Comment:
If we create a deprecation warning to the user in `__enter__`, we can just delete this note.